lambdas/search/index.py

```diff
@@ -50,23 +51,51 @@ def lambda_handler(request):
     """
 
     action = request.args.get('action')
-    indexes = request.args.get('index')
-    terminate_after = os.getenv('MAX_DOCUMENTS_PER_SHARD')
-
-    if action == 'search':
+    user_body = request.args.get('body', {})
+    user_fields = request.args.get('fields', [])
+    user_indexes = request.args.get('index', "")
+    user_size = request.args.get('size', DEFAULT_SIZE)
+    user_source = request.args.get('_source', [])
+    terminate_after = None  # see if we can skip os.getenv('MAX_DOCUMENTS_PER_SHARD')
+
+    if action == 'packages':
+        query = request.args.get('query', '')
+        body = user_body or {
+            "query": {
+                "query_string": {
+                    "analyze_wildcard": True,
+                    "lenient": True,
+                    "query": query,
+                    # see enterprise/**/bucket.py for mappings
+                    "fields": user_fields or [
+                        # package
+                        'comment', 'handle', 'handle_text^2', 'metadata_string', 'tags'
+                    ]
+                }
+            }
+        }
+        _source = user_source
+        size = user_size
+    elif action == 'search':
         query = request.args.get('query', '')
         body = {
             "query": {
-                "simple_query_string": {
+                "query_string": {
                     "analyze_wildcard": True,
+                    "lenient": True,
                     "query": query,
-                    "fields": ['content', 'comment', 'key_text', 'meta_text']
+                    # see enterprise/**/bucket.py for mappings
+                    "fields": [
+                        # object
+                        'content', 'comment', 'ext', 'key_text^2', 'meta_text',
+                        # package
+                        'comment', 'handle', 'handle_text^2', 'metadata_string', 'tags'
+                    ]
                 }
             }
         }
-        # TODO: should be user settable; we should proably forbid `content` (can be huge)
         _source = ['key', 'version_id', 'updated', 'last_modified', 'size', 'user_meta']
```

Comment:
```suggestion
        _source = ['key', 'version_id', 'updated', 'last_modified', 'size', 'user_meta', 'comment', 'handle', 'hash', 'tags', 'metadata_string', 'pointer_file']
```
@akarve this data is required to display the search cards